examples/chart/teleport-cluster/example-values/ha-on-prem.yaml

```
@@ -0,0 +1,61 @@
+# This example demonstrates how the chart can be used to deploy an on-premise Highly Available Teleport
+# using etcd as its key-value backend, and minio as S3-compatible object storage for the recordings.
+#
+# You must deploy the etcd beforehand and have a service exposing all individual etcd peers.
+# The etcd client keypair and ca certificate are stored in a `etcd-client` secret with the
+# respective names "tls.key", "tls.crt", "ca.crt".
+#
+# You must deploy minio beforehand, create an access key and store it in a secret.
+# In this example the secret is named `minio-creds` and contains two entries:
+# `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`.
+# See: https://github.com/gravitational/teleport/discussions/18499
+
+chartMode: standalone
+clusterName: teleport.example.com
+persistence:
+  enabled: false
+
+auth:
+  teleportConfig:
+    teleport:
+      storage:
+        type: etcd
+        peers: [ "https://etcd-0.etcd-headless.teleport.svc.cluster.local:2379", "https://etcd-1.etcd-headless.teleport.svc.cluster.local:2379", "https://etcd-2.etcd-headless.teleport.svc.cluster.local:2379" ]
+        prefix: /teleport/
```

Might be a bit more readable like this? I don't know which I prefer 🤷‍♂️ 

```suggestion
        type: etcd
        peers:
          - "https://etcd-0.etcd-headless.teleport.svc.cluster.local:2379"
          - "https://etcd-1.etcd-headless.teleport.svc.cluster.local:2379"
          - "https://etcd-2.etcd-headless.teleport.svc.cluster.local:2379"
        prefix: /teleport/
```